book/modules/appendix1-exercises.tex

```diff
@@ -1,7 +1,85 @@
-
 \begin{exercises}
-
 	\begin{problist}
-		\prob XXX Finish
+		\prob For each equation given below, determine if it is a linear
+		equation. If not, explain what makes it nonlinear.
+		\begin{enumerate}
+			\item $\cos(4)x_{1}+\mathrm{e}y_{2}+\pi z_{3}=\mathrm{e}^{\pi}$
+
+			\item $4x_{1}+2x_{2}+5x_{4}=4x_{2}+4x_{5}+5$
+
+			\item $12x+3xy+5z=2$
+
+			\item $\cos(4)x+\sin(4)y+\tan(4)z=\sec(4)x+\csc(4)y+\cot(4)z$
+
+			\item $5x+2y+8z=\cos(y)$
+		\end{enumerate}
+
+		\prob For each system of linear equations given below: (a) write down its
+		augmented matrix, (b) use row reduction algorithm to determine if it is
+		consistent or not, (c) for each consistent system, give its complete
+		solution.
+		\begin{enumerate}
+			\item \systeme{-10x_1-4x_2+4x_3=28, 3x_1+x_2-x_3=-8, x_1+x_2-\frac{1}{2}x_3=-3}
+
+			\item \systeme{3x_1-2x_2+4x_3=54, 5x_1-3x_2+6x_3=88, x_1=-3}
+
+			\item \systeme{x_1+2x_2+4x_3-3x_4=0, 3x_1+5x_2+6x_3-4x_4=1, 4x_1+5x_2-2x_3+3x_4=3}
+
+			\item \systeme{x_1-x_2+5x_3+x_4=1, x_1+x_2-2x_3+3x_4=3, 3x_1-x_2+8x_3+x_4=5, x_1+3x_2-9x_3+7x_4=5}
+		\end{enumerate}
+
+		\prob Consider the matrix equation $M\vec x=\vec b$, where $M=\mat{2 & 8 \\ 4 & 16}$.
+		\begin{enumerate}
+			\item Show that with $\vec b=\mat{7 \\ 14}$, the system is
```

Comment:
Maybe test the students' understanding of _consistent_ by asking a system with _one_ solution and another with _infinite_ solution. This can also test for quantifiers.